node/src/components/consensus/highway_core/state/panorama.rs

```diff
@@ -115,6 +115,15 @@ impl<C: Context> Panorama<C> {
         self.iter().filter_map(Observation::correct)
     }
 
+    /// Returns an iterator over validator indexes that are seen as faulty by this panorama.
+    pub(crate) fn iter_faulty_validators<'a>(
+        &'a self,
+    ) -> impl Iterator<Item = ValidatorIndex> + 'a {
```

Comment:
I think you can actually omit the explicit `'a` lifetime:
```suggestion
    ) -> impl Iterator<Item = ValidatorIndex> + '_ {
```